kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/MavenRepositoryConfiguration.java

```diff
@@ -155,11 +162,15 @@ private boolean isProfileActive( Profile profile ) {
                                                      .addPassword( server.getPassword() )
                                                      .build() );
         }
-        if (settings.getActiveProxy() != null) {
-            remoteBuilder.setProxy(getActiveAetherProxyFromSettings(settings));
+
+        if (activeProxy != null) {
+            if (null == activeProxy.getNonProxyHosts()) {
+                remoteBuilder.setProxy( getActiveAetherProxyFromSettings(settings));
+            } else if (!repositoryUrlMatchNonProxyHosts(settings.getActiveProxy().getNonProxyHosts(), remoteBuilder.build().getUrl())) {
+                remoteBuilder.setProxy( getActiveAetherProxyFromSettings( settings ) );
```

Comment:
```suggestion
                remoteBuilder.setProxy(getActiveAetherProxyFromSettings(settings));
```

Removing extra space